apps/feedback/models/team_feedback_form.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+import (
+	"github.com/jinzhu/gorm"
+
+	userModels "github.com/iReflect/reflect-app/apps/user/models"
+)
+
+type TeamFeedbackForm struct {
+	gorm.Model
+	Team           userModels.Team `gorm:"ForeignKey:TeamID; AssociationForeignKey:ID"`
+	TeamID         uint            `gorm:"not null"`
+	Role           userModels.Role `gorm:"ForeignKey:RoleID; AssociationForeignKey:ID"`
```

Comment:
Could be renamed to "ForRole" to further clarify the use of this field.

Reply:
done